src/com/georgidinov/Author.java

```diff
@@ -0,0 +1,95 @@
+package com.georgidinov;
+
+//        2. Създайте клас `Author`. Създайте няколко член-променливи,
+//        като прецените сами какъв тип трябва да бъдат
+//        -	`name` - името на изпълнителя
+//        -	`age` - възраст на изпълнителя
+//        -	`song` - песента на изпълнителя
+//        Напишете метод, който да извежда в конзолата форматирано
+//        името на автора и неговата възраст.
+//        Напишете метод, който да извежда в конзолата форматирано
+//        заглавието на песента и автора.
+
+import java.util.Objects;
+
+
+/**
+ * The class represents plausible song artist and author
+ * Has fields for name, age and song
+ * can be extended to hold list of songs, or album object holding songs
+ * Two chained constructors, for testing purposes
+ * the No Args constructor will default to the group metallica and the default song object
+ * (which is metallica song for now)
+ */
+public class Author {
+
+    //== fields ==
+    private String name;
+    private int age;
+    private Song song;
+
+    //== constructors ==
+    public Author() {
+        this("Metallica", 38, new Song());
+    }//end of constructor
+
+    public Author(String name, int age, Song song) {
+        this.name = Objects.requireNonNull(name, "DefaultName");
+        if (this.isValidAge(age)) {
+            this.age = age;
+        } else {
+            this.age = -1;
+        }
```

Comment:
I don't think `-1` is a valid age. It's ok for now, but usually when the input data is wrong, we throw Exceptions. We'll talk about them later